opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -334,6 +335,17 @@ CpGrid::scatterGrid(EdgeWeightMethod method,
 #else
                 OPM_THROW(std::runtime_error, "Parallel runs depend on ZOLTAN if useZoltan is true. Please install!");
 #endif // HAVE_ZOLTAN
+            }
+            else if (partitionMethod == 2)
```

Comment:
Should we make sure that partitioning always works if either METIS or Zoltan is there no matter what is requested?

Reply:
I would say no, because then the user requests a certain partitioner but another one is taken